.env

```diff
@@ -56,8 +68,5 @@ APP_EXCERPT_MAX_LENGTH=255
 
 ALLOWED_IMAGE_MIME_TYPES='["image/jpeg", "image/png"]'
 
-###> symfony/lock ###
-# Choose one of the stores below
-# postgresql+advisory://db_user:db_password@localhost/db_name
-LOCK_DSN=flock
-###< symfony/lock ###
+SITE_SEND_FROM_EMAIL=itkdev@mkb.aarhus.dk
+SITE_NAME="Det sker i Aarhus"
```

Comment:
Not sure that should be default site name, as project should work for whole midt-jylland...

Reply:
Also missing new-line at end of file

Reply:
😆